pkg/handler/collector/collector_test.go

```diff
@@ -168,7 +150,7 @@ func testSubscribe(ctx context.Context, docChannel chan<- processor.DocumentTree
 		if err != nil {
 			fmtErr := fmt.Errorf("[processor: %s] failed unmarshal the document bytes: %v", id, err)
```

Comment:
just noticing this, but if its wrapping an error should be "%w".

Reply:
Ill add this quick fix in the other PR that updates NATs for the de-duplication.